### ISSUE TITLE
starvote format shouldn't permit specifying the same candidate twice on one ballot

### ISSUE THREAD

**Issue by masiarek** [larryhastings/starvote#8]
[options]

    method = star
    tiebreaker = none
    verbosity = 1

[ballots]

    Andre  = 3
    Andre  = 1
    Blake  = 0
    Carmen = 0
    
    Blake  = 0
    Carmen = 0

    Andre  = 0
    Blake  = 0
    Carmen = 0    

```
[STAR Voting]
  Tabulating 3 ballots.
  Maximum score is 5.
[STAR Voting: Scoring Round]
  The two highest-scoring candidates advance to the next round.
    Andre  -- 1 (average 1/3) -- First place
    Blake  -- 0 (average 0)   -- Tied for second place
    Carmen -- 0 (average 0)   -- Tied for second place
  Andre advances, but there's a two-way tie for second.
[STAR Voting: Scoring Round: First tiebreaker]
  The candidate preferred in the most head-to-head matchups advances.
    Blake         -- 0 -- Tied for second place
    Carmen        -- 0 -- Tied for second place
    No Preference -- 3
  There's still a two-way tie for second.
[STAR Voting: Scoring Round: Second tiebreaker]
  The candidate with the most votes of score 5 advances.
    Blake  -- 0 -- Tied for second place
    Carmen -- 0 -- Tied for second place
  There's still a two-way tie for second.
[STAR Voting: Scoring Round: Unbreakable Tie]
  Tie between Blake and Carmen.
```

**Comment by larryhastings** [comment]
What do you claim is the error?  An individual ballot shouldn't have all zeroes?

**Comment by masiarek** [comment]
The first ballot is incorrect- it has Andre twice - this should be flagged by some validation program- before tabulating (similar to catching max score).

**Comment by larryhastings** [comment]
Ah, good catch.  Fixed in 2.1.1, will be released momentarily.
